Extension/package.json

```diff
@@ -2333,22 +2333,22 @@
           "markdownDescription": "%c_cpp.configuration.simplifyStructuredComments.markdownDescription%",
           "scope": "application"
         },
-        "C_Cpp.autocompleteDoxygenComment":{
-          "type":"boolean",
-          "default":true,
-          "description": "%c_cpp.configuration.autocompleteDoxygenComment.description%",
+        "C_Cpp.doxygen.generateCommentsOnTyping":{
```

Comment:
onType is a more or less common setting/command name (e.g. formatOnType) which is why I suggested `generateOnType`.  I was hoping for shorter names, and I don't think "comments" is needed because 'doxygen' is synonymous with 'comments'

Reply:
Though I suppose I somewhat contradict that point by suggesting commentStyle.  So maybe I just like it shorter in this case and not the other. 🤷‍♂️